tools/make-win64-dev-env.bat

```diff
@@ -1,5 +1,5 @@
 @echo off
-%SystemRoot%\System32\WindowsPowerShell\v1.0\powershell.exe -NoProfile -ExecutionPolicy Bypass .\tools\provision.ps1
```

Comment:
We could move the responsibility of determining the current location of execution into the `provision.ps1` script, and this might alleviate some confusion about the intended uses of these scripts. Either that or we could possibly introduce a warning if the script isn't being run from the repo root.